src/components/interestedPeople/DropdownPagnation.tsx

```diff
@@ -0,0 +1,87 @@
+"use client"
+import React, { useState, useRef, useEffect } from 'react';
+import { CgChevronDown } from 'react-icons/cg';
+import Pagination from '../Pagination';
+import { DropdownPaginationProps } from '@/types/table';
+
+const DropdownPagination: React.FC<DropdownPaginationProps> = ({
```

Comment:
Pass all this to a custom hook and call it here
This way it looks more organized and there is only the html